src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +163,180 @@ export default async function() {
+
+Astro DB can connect to any local libSQL database or to any server that exposes the libSQL remote protocol, whether managed or self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL DB
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server. This is required for remote databases, not needed for local DBs like files or in-memory databases
+
+The way you get these values varies between services, but the next section will show you how you can get started with [Turso](https://turso.tech).
+
+Turso is the company behind [libSQL](https://github.com/tursodatabase/libsql), the open-source fork of SQLite that powers Astro DB. They created and maintain the libSQL project, making them the perfect choice to provide a fully managed libSQL database platform.
+
+### Getting started with Turso
+
+You can either use the CLI or the web interface to create and retrieve the values you need.
+The steps that are shown below will walk you through the process of installing the Turso CLI, logging in (or signing up), creating a new database, getting the two values we need and pushing our schema to the remote database.
+
+<Steps>
+
+1. Install the [Turso CLI](https://docs.turso.tech/cli/installation).
+
+2. [Login or signup](https://docs.turso.tech/cli/authentication) to Turso.
+
+3. Next, create a new database. In this example the database name is `andromeda`, but you can change it to any other name.
+
+   ```sh "andromeda"
+   turso db create andromeda
+   ```
+
+4. You can see the information about the newly created DB with the following command:
+
+   ```sh "andromeda"
+   turso db show andromeda
+   ```
+
+   Use the value from the row `URL:` as the environment variable `ASTRO_DB_REMOTE_URL`.
+   
+   :::tip
+   To read more about .env files and environment variables, check out the [Using environment variables](/en/guides/environment-variables/) guide.
+   :::
+
+   ```env title=".env" "libsql://andromeda-houston.turso.io"
+   ASTRO_DB_REMOTE_URL=libsql://andromeda-houston.turso.io
+   ```
+
+5. To authenticate requests to the database, create a new token for it:
+
+   ```sh "andromeda"
+   turso db tokens create andromeda
+   ```
+
+   Use the output of the command as the environment variable `ASTRO_DB_APP_TOKEN`
+
+   ```env title=".env" add={2} "eyJhbGciOiJF...3ahJpTkKDw"
+   ASTRO_DB_REMOTE_URL=libsql://andromeda-houston.turso.io
+   ASTRO_DB_APP_TOKEN=eyJhbGciOiJF...3ahJpTkKDw
+   ```
+
+6. Push your DB schema and metadata to the new Turso database.
+
+   ```sh
+   astro db push --remote
+   ```
+
+7. Congratulations, now you have a database connected! Give yourself a break. 👾
+
+   ```sh
+   turso relax
+   ```
+
+</Steps>
+
+To explore more features of Turso, check out the [Turso docs](https://docs.turso.tech).
+
+This was a quick guide on how to get started with Turso, read on to learn more on how to use Astro DB.
+
```

Comment:
While I understand the intent, I'm not sure this sentence fits in with the rest of the documentation. I think that by deleting it it doesn't take anything away from the documentation and those who get there see clearly that the documentation continues.

```suggestion
```